### PR TITLE
Add containerLogsDir to store legacy symbolic

### LINKED ISSUE #1883: Failed to create legacy symbolic
**What happened**:
edgecore continuely logs `Failed to create legacy symbolic link` Error for unknown reason.
image
**What you expected to happen**:
stop logging that error and figure out why.
**How to reproduce it (as minimally and precisely as possible)**:
run edgecore
**Anything else we need to know?**:

**Environment**:
- KubeEdge version(e.g. `cloudcore/edgecore --version`):

### PR DESCRIPTION
**Pull request by Poor12**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

/kind bug

**What this PR does / why we need it**:

Fixes #1883 

**Special notes for your reviewer**:

```release-note

```

### CODE REVIEW

File: edge/pkg/edged/edged.go [hunk 1]
```diff
@@ -680,7 +682,12 @@ func (e *edged) initializeModules() error {
 		klog.Errorf("Failed to initialNode %v", err)
 		return err
 	}
-
+	// If the container logs directory does not exist, create it.
+	if _, err := os.Stat(ContainerLogsDir); err != nil {
+		if err := e.os.MkdirAll(ContainerLogsDir, 0755); err != nil {
+			klog.Errorf("Failed to create directory %q: %v", ContainerLogsDir, err)
```
Comment (daixiang0) [inline edge/pkg/edged/edged.go:688]: Here %s is enough, see https://gobyexample.com/string-formatting.

Comment (fisherxu) [reply edge/pkg/edged/edged.go:688]: Err is not string here, why use %s? I think %v is consistent with k8s

Comment (daixiang0) [reply edge/pkg/edged/edged.go:688]: I mean no need to use %q.

Comment (Poor12) [reply edge/pkg/edged/edged.go:688]: It seems the only difference is whether the output contains double quotes.k8s use %q, I think keeping the same is no problem.

**Pull request merged**